app/scripts/app.js

```diff
@@ -53,14 +54,19 @@ app.config(function($routeProvider) {
       templateUrl: 'views/confirm-order.html',
       controller: 'OrderController'
     })
+    .when('/payment/:id', {
+      templateUrl: 'views/payment.html',
+      controller: 'PaymentController'
+    })
     .otherwise({
       redirectTo: '/'
     });
 });
 
 app.constant('CONFIG', {
-  // 'BASE_URI': '//localhost:3000',
-  'BASE_URI': '//tma-develop.herokuapp.com',
+  'BASE_URI': '//localhost:3000',
```

Comment:
never used ENV var before with angular, but I just found this article => http://mindthecode.com/how-to-use-environment-variables-in-your-angular-application

Will give it a try :wink: